sdk/test-utils/recorder/test/utils.spec.ts

```diff
@@ -422,4 +427,265 @@ ultramarine.com/url/PUBLIC
+      const result = generateTestRecordingFilePath(platform, testSuiteTitle, testTitle);
+      expect(result).to.equal(
+        `${platform}/utils_generatetestrecordingfilepath/recording_should_generate_a_properly_formatted_path_on_platform_node.js`
+      );
+    });
+
+    it("should generate a properly formatted path on platform: Browsers", function() {
+      const platform = "browsers";
+      const testSuiteTitle = this.test!.parent!.fullTitle();
+      const testTitle = this.test!.title;
+      const result = generateTestRecordingFilePath(platform, testSuiteTitle, testTitle);
+      expect(result).to.equal(
+        `${platform}/utils_generatetestrecordingfilepath/recording_should_generate_a_properly_formatted_path_on_platform_browsers.json`
+      );
+    });
+  });
+
+  describe("nodeRequireRecordingIfExists", () => {
+    it("should be able to load the contents of a recording file if the file exists", function() {
+      if (isBrowser()) return this.skip();
+
+      const mockFs = require("mock-fs");
+      const mockRequire = require("mock-require");
+
+      // This needs to change if findRecordingsFolderPath changes.
+      mockFs({
+        // Our lazy require doesn't use the fs module internally.
+        "recordings/recording.json": "",
+        "test/myTest.spec.ts": "",
+        "../../sdk/": {},
+        "../../../rush.json": ""
+      });
+
+      mockRequire("../recordings/recording.json", {
+        property: "value"
+      });
+
+      const path = require("path");
+      const testAbsolutePath = path.resolve("test/myTest.spec.ts");
+
+      expect(nodeRequireRecordingIfExists("recording.json", testAbsolutePath).property).to.equal(
+        "value"
+      );
+
+      mockFs.restore();
+      mockRequire.stopAll();
+    });
+
+    it("should throw if the file at a given recording path doesn't exist", function() {
+      if (isBrowser()) return this.skip();
+
+      // Require shouldn't be mocked in this test since we should be preventing require from being reached.
+
+      const mockFs = require("mock-fs");
+
+      // This needs to change if findRecordingsFolderPath changes.
+      mockFs({
+        recordings: {},
+        "test/myTest.spec.ts": "",
+        "../../sdk/": {},
+        "../../../rush.json": ""
+      });
+
+      const path = require("path");
+      const testAbsolutePath = path.resolve("test/myTest.spec.ts");
+
+      let error: Error | undefined;
+
+      try {
+        nodeRequireRecordingIfExists("recording.json", testAbsolutePath);
+      } catch (e) {
+        error = e;
+      }
+
+      expect(error!.message).to.equal(
+        `The recording recording.json was not found in ${findRecordingsFolderPath(
+          "recording.json"
+        )}`
+      );
+
+      mockFs.restore();
+    });
+  });
+
+  describe("testHasChanged", () => {
+    it("In NodeJS - should not crash if the recorded file doesn't exist", function() {
+      if (isBrowser()) return this.skip();
```

Comment:
Move the tests meant for the node to `test/node` folder and update the rollup.test config accordingly, to avoid 
 unnecessary `this.skip()` here.